elasticsearch5/search-index/src/main/java/org/vertexium/elasticsearch5/Elasticsearch5SearchIndex.java

```diff
@@ -431,33 +471,29 @@ void addElement(
 
         bulkUpdateService.flushUntilElementIdIsComplete(elementMutation.getId());
```

Comment:
We should wait in the BulkUpdateService when the batch is converted to a request.